pageserver/src/http/routes.rs

```diff
@@ -179,20 +196,23 @@ async fn timeline_create_handler(mut request: Request<Body>) -> Result<Response<
         .new_timeline_id
         .unwrap_or_else(TimelineId::generate);
 
+    let ctx = RequestContext::new(TaskKind::MgmtRequest, DownloadBehavior::Download);
```

Comment:
Not sure I fully understand why `timeline_create` can download layers. Should it be `Error` instead?

Reply:
Hmm, agreed, timeline creation shouldn't require downloading anything. Let's make it `Error`, or `Warn` if we're note quite sure.